.cfignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
+.ruby-version
+#test
```

Comment:
this file also doesn't need to be in the PR. It's just for cloud foundry hosting. It won't hurt anything, but it's unneccessary.